pom.xml

```
@@ -63,6 +63,19 @@
       <version>2.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins</groupId>
+      <artifactId>configuration-as-code</artifactId>
+      <version>1.29</version>
```

There is a new version fixing a small bug in testing
```suggestion
      <version>1.30</version>
```